controllers/front/IdentityController.php

```diff
@@ -31,6 +31,8 @@ class IdentityControllerCore extends FrontController
     public $authRedirection = 'identity';
     public $ssl = true;
 
+    public $passwordRequired = true;
```

Comment:
Maybe this property should be protected in order to reduce risk of changing this property from another piece of code.

Reply:
Can't see why not, our test case is to add a hook to this class :)
Tested and approved, Would you amend your commit since you're last in line?